pkg/DWSUtility.py

```diff
@@ -59,12 +59,6 @@ class DWSUtility:
         "nnfnodestorages.nnf.cray.hpe.com",
         "nnfstorageprofiles.nnf.cray.hpe.com",
         "nnfstorages.nnf.cray.hpe.com",
-        "rsyncnodedatamovements.dm.cray.hpe.com",
-        "rsynctemplates.dm.cray.hpe.com",
-    ]
-
-    NON_HPE_CRDS = [
-        ["mpijobs.kubeflow.org", "v1"],
```

Comment:
Doesn't lustre-to-lustre still use this?

Reply:
Not for data movement v2 - if you need it in the meanwhile I can put it back; but I was anticipating changes for v2